spec/models/application_choice_spec.rb

```diff
@@ -433,4 +433,21 @@
       end
     end
   end
+
+  describe '#science_gcse_needed?' do
+    it 'is true for primary courses' do
+      course = create(:course)
+      course_option = create(:course_option, course:)
+      application_choice = create(:application_choice, course_option:)
+      expect(application_choice.science_gcse_needed?).to be true
```

Comment:
where is the primary defined? Default? I am not sure if others agreed but I'd prefer to be explicit here 🤔 . Wdyt?

Reply:
Course is set to primary by default

Reply:
but perhaps good to be explicit when i think about those who might be reading the spec for the first time